RecoBTag/PixelCluster/python/pixelClusterTagInfos_cfi.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+pixelClusterTagInfos = cms.EDProducer("PixelClusterTagInfoProducer",
```

Comment:
this file is redundant, mostly.
```isPhase1``` should be set appropriately via an era

Reply:
We added the era modifier from Configuration/Eras/python/Modifier_pixel_2016_cff